groups/instance/locals.tf

```diff
@@ -66,5 +66,5 @@ locals {
 
   admin_password                             = local.secrets.admin_password
 
-  efs_kms_key_id                             = local.security_efs_kms_keys_data.ebs
+  efs_access_point_id                        = module.efs_file_system.efs_access_point_ids["${var.efs_artifacts_access_point_name}"].id
```

Comment:
As there is whitespace between this and the previous local var, this resets the need to align the arguments.

E.g.:
```
  efs_access_point_id = module.efs_file_system.efs_access_point_ids["${var.efs_artifacts_access_point_name}"].id
```

It would be beneficial to run `terraform fmt` to review/update formatting. I would recommend running it with `-check -diff` initially so any formatting updates can be reviewed.

Reply:
Completed as required